### ISSUE TITLE
fix: typo on blogs

### ISSUE THREAD

**Issue by adarshaacharya** [adarshaacharya/portfolio-v2#79]
- since the site has [blogs section](https://adarshaacharya.com.np/blog/) , and recently i found growth in audience viewing the blog, but i found that there are so many typos on my blog posts.

- So, on occasion of **Hacktoberfest** , if you can find some typos on my blog and send PR fixing it .. it might be good point if you want to get started with open source. 

- Don't forget to read [contributing guidelines](https://github.com/adarshaacharya/adarshaacharya.com.np/blob/master/CONTRIBUTING.md) for getting started.. Also you can comment down below if you face any sort of problems.

Good luck!!

**Comment by Ln11211** [comment]
@adarshaacharya, Can I try this ?, please assign this to me

**Comment by adarshaacharya** [comment]
@Ln11211  Sure !!

**Comment by Ln11211** [comment]
@adarshaacharya, before I start working on the fixing the blog, can you tell me how do I do it ?, should i fork and clone the project ?
also, where can I find the blog content to proof read and edit the typos ?

**Comment by adarshaacharya** [comment]
Yeah you shoul fork and clone the project. Then inside [content/posts](https://github.com/adarshaacharya/adarshaacharya.com.np/tree/master/content/posts) you can find the contents of the blogs serially.
The content are written on markdown. I recommend you reading blog from https://adarshaacharya.com.np/blog/ , if  you find any error go ahead and change the content. 

If you are just fixing typos then you can directly edit it from github by forking the repo without cloning it on your PC. This will make process faster. 

Also while commiting your changes use meaningful commits and make pull request with meaningful title. 

You can read how to write PR title [here.](https://www.gatsbyjs.org/contributing/managing-pull-requests/#commit-and-pr-title)

Hope that it helps you.

**Comment by Ln11211** [comment]
@adarshaacharya, I am just going to try and fix typos, I'll fork it and do it in github.., thank you..
